### DISCUSSION TITLE
¿Actualizaciones?

### DISCUSSION THREAD

**Discussion by ElizabethLeonPerez** [Launch-X-Latam/MisionFrontEnd#600] [Team Posts]
Hola!! ¿Alguien sabe cómo sigue esto? ¿Se va a hacer el hackathon? ¿Y los certificados? En el teams nunca me volvió a llegar una notificación y veo que por acá tampoco hay mucha info.

**Comment by mabvmex** [comment]
Parece que no habrá más… Ya estamos a mediados de Marzo cuando se habló de continuar en Enero… Parece tiempo de afrontar que terminó así, sin más.

**Comment by DiegoCorchuelo** [comment]


**Reply by Mauricio-Mizhquiri** [reply]
A parte que no pagaste nada vienes a estarte quejando, dedícate a estudiar y no armar lio por cosas innecesarias

**Comment by JehuJmz** [comment]
Personalmente no comparto esa opinión, los cursos, los contenidos y el conocimiento compartido es muy bueno y de calidad, y creo que con el soporte y apoyo que ellos brindan, no nos quita nada darles la misma comprensión, después de todo las grabaciones y los accesos ahí siguen para que los aprovechen.

**Reply by xramons** [reply]
Es importante tener en cuenta que **la calidad del aprendizaje no se basa solo en el contenido, sino también en la experiencia emocional y personal del estudiante**. Si alguien se siente frustrado o decepcionado por una promesa que no se cumplió, puede afectar negativamente su motivación y compromiso con el aprendizaje.

Si bien puede ser comprensible que se produzcan desafíos imprevistos durante el proceso de desarrollo, es importante que las plataformas sean claras acerca de cualquier cambio para que los estudiantes puedan ajustar sus expectativas y tomar decisiones informadas acerca de su aprendizaje.

**Comment by Drako9159** [comment]
Creo que lo que estuvo mal fue la parte del backend, se notó muy apresurada y con falta de contenido, estuve en la primera versión de este programa que fue lanzado creo que solo en México, fue una muy buena experiencia, ahí sí se realizó un hackatón, pero lo de los certificados de participación nunca se contempló, solo nos quedaba esa insignia en github que nos deja ver como participantes.

**Comment by Drako9159** [comment]
Ahora, no quisiera hacer esta comparación, pero el bootcamp de Oracle me está pareciendo muy bueno, también es gratuito y ahí sí les dan esas insignias de participación, aprenden a programar y se enfocan en frontend o backend con java, es un muy buen programa.

**Comment by NelsonOrtizLeal** [comment]
Esto de acuerdo, es buen contenido de launch X, además aún están en la plataforma para verlo. Cómo dicen los demás, lo que importa es que muevas las manos y realices los proyectos. Creo que vale más que tengas los proyectos y las personas vean que sabes trabajar.

A qué Bootcamp de Oracle te refieres @Drako9159, aún tienes el link para que me lo compartas.

**Comment by Drako9159** [comment]
@NelsonOrtizLeal este es el programa, tienen inscripciones abiertas hasta el 27 de marzo [Oracle](https://www.oracle.com/mx/education/oracle-next-education/)

**Comment by NelsonOrtizLeal** [comment]
Gracias @Drako9159 .

**Comment by FedericoF03** [comment]
Para los que se están quejando, esta bien quejarse si un servicio AUNQUE GRATIS no cumple con lo dicho, no hay porque atacarlos por decir la verdad, también recuerden que no todos entraron por la misma razón algunos solo queríamos el certificado marcado por Microsoft para sumar al CV. Básicamente al no cumplir lo dicho hicieron perder el tiempo a muchas personas.
